apps/comments/src/components/Comment.vue

```diff
@@ -76,6 +76,8 @@
 					<NcRichContenteditable ref="editor"
 						:auto-complete="autoComplete"
 						:contenteditable="!loading"
+						:label="editor ? t('comments', 'Write new comment') : t('comments', 'Edit comment')"
+						:placeholder="t('comments', 'Write a comment…')"
```

Comment:
Not 100% sure, but 90% sure, especially with "write" and a space before `…`.

```suggestion
						:label="editor ? t('comments', 'Write a new comment') : t('comments', 'Edit a comment')"
						:placeholder="t('comments', 'Write a comment …')"
```

Reply:
Done :white_check_mark:

Reply:
Article is not needed in this case?